bgpd/bgp_vty.c

```diff
@@ -6588,6 +6588,62 @@ DEFPY (bgp_imexport_vpn,
 	return CMD_SUCCESS;
 }
 
+DEFPY (af_routetarget_import,
+       af_routetarget_import_cmd,
+       "[no] <rt|route-target> redirect import RTLIST...",
+       NO_STR
+       "Specify route target list\n"
+       "Specify route target list\n"
+       "For redirected traffic matching that import rt\n"
+       "Import routes to this address-family\n"
```

Comment:
I don't see an argument that matches this string.  Am I missing something?

Reply:
this matches  import keyword

```r1(config-router-af)# rt  ?
  redirect  For redirected traffic matching that import rt
  vpn       Between current address-family and vpn
r1(config-router-af)# rt redirect ?
  import  Import routes to this address-family
r1(config-router-af)# rt redirect import ?
  RTLIST  Space separated route target list (A.B.C.D:MN|EF:OPQR|GHJK:MN)
```